src/composeappmanager.h

```diff
@@ -30,15 +30,10 @@ class ComposeAppManager : public OstreeManager {
   data::InstallationResult install(const Uptane::Target& target) const override;
   std::string name() const override { return PACKAGE_MANAGER_COMPOSEAPP; };
 
- private:
-  // TODO: consider avoiding it
-  FRIEND_TEST(ComposeApp, getApps);
-  FRIEND_TEST(ComposeApp, handleRemovedApps);
-  FRIEND_TEST(ComposeApp, fetch);
```

Comment:
this is so obvious now. i don't know why i didn't do it this way to begin with :)